gamepatches.py

```diff
@@ -3286,3 +3319,16 @@ def do_patch_custom_dowsing_images(self):
         ).read_bytes()
         arc.set_file_data("timg/tr_dauzTarget_18.tpl", sandshipdata)
         do_button_path.write_bytes(arc.to_buffer())
+
+    def patch_random_starting_statue_flags(self):
+        for statue in self.placement_file.start_statues.values():
+            flag = BIRD_STATUE_FLAGS.get(statue[1], None)
```

Comment:
```suggestion
            flag = BIRD_STATUE_FLAGS.get(statue[1])
```
The None is redundant